x-pack/test/cases_api_integration/security_and_spaces/tests/common/cases/get_metrics.ts

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
Sorry about all the conflicts 😬  I think we can delete this file, it has been moved to here: https://github.com/elastic/kibana/blob/main/x-pack/test/cases_api_integration/security_and_spaces/tests/common/metrics/get_case_metrics.ts